pkg/metrics/metrics.go

```diff
@@ -129,6 +129,35 @@ var (
 		Help:        "Last timestamp when we received an event",
 		ConstLabels: prometheus.Labels{"source": LabelEventSourceAPI},
 	})
+
+	// Proxy statistics
+	// ProxyParseErrors is a count of failed parse errors on proxy
+	ProxyParseErrors = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Name:      "proxy_parse_errors",
```

Comment:
Please follow Prometheus's naming conventions:
https://prometheus.io/docs/practices/naming/
For instance, I'd suggest naming those metrics:
```
proxy_parse_errors_total
proxy_forwarded_requests_total
proxy_denied_requests_total
proxy_received_requests_total
```

Reply:
done